build/ci/linux/build.sh

```diff
@@ -1,4 +1,25 @@
 #!/usr/bin/env bash
+##
+## // SPDX-License-Identifier: GPL-3.0-only
+## // MuseScore-CLA-applies
```

Comment:
Do we have to indicate these two lines? Should they be above the line? I would like to see them below the line, if they are required, it's easier to perceive

Reply:
These two lines are not necessary if the main text is present, but equally the main text is not necessary if these lines are present. So my preference would be to remove the main text and keep just the following:

```
#!/usr/bin/env shell
# SPDX-License-Identifier: GPL-3.0-only
# MuseScore-CLA-applies
#
# MuseScore
# Music Composition & Notation
#
# Copyright (C) 2021 MuseScore BVBA and others
```

The SPDX license identifier is an industry standard shorthand for the license in question (see https://spdx.org/licenses/).

This is how the [Linux kernel does it](https://github.com/torvalds/linux/blob/master/kernel/audit.c#L1-L9) for example. The kernel licensing rules page says (just above [here](https://www.kernel.org/doc/html/latest/process/license-rules.html#license-identifier-syntax)):

> SPDX license identifiers are machine parsable and precise shorthands for the license under which the content of the file is contributed. SPDX license identifiers are managed by the SPDX Workgroup at the Linux Foundation and **have been agreed on by partners throughout the industry, tool vendors, and legal teams**.

There is no need to state the full disclaimer because that is already part of the GPL itself. Even the link to the GPL is not needed because these days it is a widely recognised license and an internet search for "GPL 3" gives the correct result (for example, it has been [confirmed in a US court](https://en.wikipedia.org/wiki/Creative_Commons_license#Drauglis_v._Kappa_Map_Group,_LLC) that "CC BY-SA 2.0" without a link is sufficient text to identify the [Creative Commons Attribution-ShareAlike 2.0 license](https://creativecommons.org/licenses/by-sa/2.0/)).